src/cluster.c

```diff
@@ -4911,10 +4898,10 @@ void addReplyClusterLinkDescription(client *c, clusterLink *link) {
     addReplyBulkCString(c, events);
 
     addReplyBulkCString(c, "send-buffer-allocated");
-    addReplyLongLong(c, sdsalloc(link->sndbuf));
+    addReplyLongLong(c, link->send_msg_queue_mem);
 
     addReplyBulkCString(c, "send-buffer-used");
-    addReplyLongLong(c, sdslen(link->sndbuf));
+    addReplyLongLong(c, link->send_msg_queue_mem);
```

Comment:
now send-buffer-allocated and send-buffer-used are the same, do we have plans to remove one or update the doc? https://redis.io/commands/cluster-links/

> send-buffer-allocated: Allocated size of the link's send buffer, which is used to buffer outgoing messages toward the peer.
send-buffer-used: Size of the portion of the link's send buffer that is currently holding data(messages).

Reply:
We kept it to prevent a "breaking" change. We could update the documentation. This was always mostly used as a debugging command.